lib/inv_gcr_quda.cpp

```diff
@@ -38,6 +38,7 @@ namespace quda {
 
     inner.inv_type_precondition = QUDA_INVALID_INVERTER;
     inner.is_preconditioner = true; // tell inner solver it is a preconditioner
+    inner.pipeline = true;
```

Comment:
Why was this change made?

Reply:
@maddyscientist

Reply:
To accelerate the preconditioner.  Robustness is not a problem for a preconditioner since it will only run for a handful of iterations.